src/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -641,6 +640,9 @@
   <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp'">
     <Reference Include="System.IO.Compression.Brotli" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' != 'uap'">
+    <Reference Include="System.Security.Cryptography.OpenSsl" />
```

Comment:
Hm.  Does it work in the TargetsUnix section?  That should be the only place it's needed.

Reply:
Moved it, let's see what CI says.